ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-operator__v4.13-aws-osd.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  ocp-addons-operators-cli:
+    name: ocp-addons-operators-cli
+    namespace: ci
+    tag: latest
+  upi-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
+      version: "4.13"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: qetest-operator-aws-osd
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      COMPUTE_NODES: "6"
+      OPENSHIFT_VERSION: "4.13"
+      OPERATOR1_CONFIG: name=rhods-operator;namespace=redhat-ods-operator
+    workflow: operator-aws-osd
```

Comment:
no test is running, why do we need this?

Reply:
it's for sanity check on osd operators flow